src/CSVerifier.sol

```diff
@@ -187,6 +187,63 @@ contract CSVerifier is ICSVerifier, AccessControlEnumerable, PausableUntil {
         _pauseFor(duration);
     }
 
+    /// @inheritdoc ICSVerifier
+    function processSlashedProof(
+        ProcessSlashedInput calldata data
+    ) external whenResumed {
+        if (data.recentBlock.header.slot < FIRST_SUPPORTED_SLOT) {
+            revert UnsupportedSlot(data.recentBlock.header.slot);
+        }
+
+        {
+            bytes32 trustedHeaderRoot = _getParentBlockRoot(
+                data.recentBlock.rootsTimestamp
+            );
+            if (trustedHeaderRoot != data.recentBlock.header.hashTreeRoot()) {
+                revert InvalidBlockHeader();
+            }
+        }
+
+        if (!data.validator.object.slashed) {
+            revert ValidatorIsNotSlashed();
+        }
+
+        // TODO: Consider uncommenting or removing the block.
+        // if (
+        //     _computeEpochAtSlot(data.recentBlock.header.slot) <
+        //     data.validator.object.withdrawableEpoch
+        // ) {
+        //     revert ValidatorIsNotWithdrawable();
+        // }
```

Comment:
Would suggest to have this block to prevent unexpected `submitWithdrawals` with `slashingPenalty` before possible withdrawal.

Reply:
So far it doesn't add anything to security other than time constraints. But it might be reconsidered later.